Casks/rider-eap.rb

```diff
@@ -23,6 +23,7 @@
   depends_on macos: ">= :high_sierra"
 
   app "Rider EAP.app"
+  binary "#{appdir}/Rider EAP.app/Contents/MacOS/rider", target: "rider-eap"
```

Comment:
This formula does not put the version into the app name like the others. Not sure if intentional or not.

Can fix in a follow-up if desired.

Reply:
It's intended.  Nevermind!